src/hotspot/share/gc/epsilon/epsilonHeap.cpp

```diff
@@ -125,17 +125,18 @@ HeapWord* EpsilonHeap::allocate_work(size_t size, bool verbose) {
 
       // Expand and loop back if space is available
       size_t size_in_bytes = size * HeapWordSize;
-      size_t space_left = max_capacity() - capacity();
+      size_t uncommitted_space = max_capacity() - capacity();
+      size_t unused_space = max_capacity() - used();
```

Comment:
Please also: 

```
assert(unused_space <= uncommitted_space, 
       "Unused (" SIZE_FORMAT ") <= uncommitted (" SIZE_FORMAT ")",
       unused_space, uncommitted_space);
```

Reply:
I think it should be `assert(unused_space >= uncommitted_space, ...)`.

Reply:
Right!